dist.sh

```diff
@@ -2,15 +2,14 @@
 
 # 1. commit to bump the version and update the changelog/readme
 # 2. tag that commit
-# 3. use dist.sh to produce tar.gz for all platforms
+# 3. use dist.sh to produce tar.gz for all platforms and docker push nsqio/nsq
 # 4. upload *.tar.gz to bitly s3 bucket
-# 5. docker push nsqio/nsq
-# 6. push to nsqio/master
-# 7. update the release metadata on github / upload the binaries there too
-# 8. update nsqio/nsqio.github.io/_posts/2014-03-01-installing.md
-# 9. send release announcement emails
-# 10. update IRC channel topic
-# 11. tweet
+# 5. push to nsqio/master
+# 6. update the release metadata on github / upload the binaries there too
+# 7. update nsqio/nsqio.github.io/_posts/2014-03-01-installing.md
+# 8. send release announcement emails
+# 9. update IRC channel topic
+# 10. tweet
```

Comment:
Can we revert these superfluous text changes in the doc?

Reply:
👍 they are now gone.